python/S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0",
+  "Parameters": {
+    "RuleName": "S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS",
+    "SourceRuntime": "python2.7",
+    "CodeKey": "S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS.zip",
+    "InputParameters": "{\"BlockPublicAcls\": \"Off\"}",
+    "OptionalParameters": "{}",
```

Comment:
Add all the 4 parameters with an empty value here.

Reply:
Done